ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -300,16 +280,51 @@ class CommandTrackerFlowTest
       }
 
       "timeout the command when the MRT passes" in {
+        val Handle(submissions, results, _, _) = runCommandTrackingFlow(allSubmissionsSuccessful)
+
+        submissions.sendNext(newSubmission(commandId, dedupTime = Some(Duration.ofMillis(500))))
```

Comment:
How long do these new tests take? Do I understand correctly that at least 2.5 s? Are we fine with that?

Reply:
Yeah, about that. I think it's reasonable given that I don't want to rewrite this whole tracking flow mechanism. I don't understand it well enough to refactor it a lot.

Reply:
I cut this down as much as I could. As these are async tests I'm very hesitant to reduce the timeouts further, as very short timeouts might cause flaky CI runs.